### DISCUSSION TITLE
Partitioned Tables

### DISCUSSION THREAD

**Discussion by alandepaz** [toluaina/pgsync#408] [Q&A]
Can I use this with partitioned tables?  I set up my schema to read from a partitioned table, and it seems to be reading the changes and putting them on redis but i don't see the changes reflected on elastic. Is there a config I am missing or are they not supported?

**Comment by toluaina** [comment]
Were you able to run the bootstrap successfully? If so you should be able to see the changes propagated to Elasticsearch.
Can you provide more details about your setup? e.g Postgres/Elastic version etc
